### PR TITLE
MM-33310 "Message" in status update modal accepts empty character string

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Trimming and testing the submitted message, rejecting the interactive dialog request if the message is empty
- e2e tests

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-33310

### CODE REVIEW

File: tests-e2e/cypress/integration/frontstage/rhs/latest_update_spec.js [hunk 1]
```diff
@@ -82,37 +82,38 @@ describe('incident rhs > latest update', () => {
...
+                        cy.executeSlashCommand('/incident update');
 
-                    // * Verify that the interactive dialog contains a generic message
-                    cy.get('#interactiveDialogModal').within(() => {
-                        cy.get('#interactiveDialogModalIntroductionText')
-                            .contains('Update your incident status. This post will be broadcasted to a private channel.');
+                        // * Verify that the interactive dialog contains a generic message
+                        cy.get('#interactiveDialogModal').within(() => {
+                            cy.get('#interactiveDialogModalIntroductionText')
+                                .contains('Update your incident status. This post will be broadcasted to a private channel.');
```
Comment (cpoile) [inline tests-e2e/cypress/integration/frontstage/rhs/latest_update_spec.js:113]: My ide correcting formatting, sorry!

**Review by crspeller — APPROVED** [review]
Is there a ticket to fix this webapp side?

**Comment by cpoile** [comment]
@crspeller There is now: https://mattermost.atlassian.net/browse/MM-33562

**Review by agarciamontoro — APPROVED** [review]
Nice, thanks!

**Review by prapti — APPROVED** [review]
Tested and verified. Looks good!
Thank you for updating the tests @cpoile!

**Pull request merged**